app/src/main/java/com/example/music/player/view/songs/SongManager.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.music.player.view.songs
+
+import android.content.Context
+import android.media.AudioAttributes
+import android.media.MediaPlayer
+import com.example.music.player.model.entity.Song
+
+class SongManager(private val context: Context) {
```

Comment:
You can remove context from constructor and add it to playSong method as parameter

Reply:
This class should be Singleton